AndelaOSP/Andela-Socials
# #163719849 Fix get calendar invite when new event is created in a category

#### What Does This PR Do?
This fixes the failure to send calendar invites to the other users of a category(group) everytime a new event is created in a category(group).
#### Description Of Task To Be Completed
1. Modify the ```send_calendar_invites``` async function found in ```/server/graphql_schemas/utils/helpers.py``` with the ability to get a correct invitee_list
2. Upgrade relevant packages related to the google calendar API by running the following with an active virtual environment
   - pip install --upgrade google-api-python-client
   - pip install --upgrade google-auth google-auth-oauthlib google-auth-httplib2
   - pip install --upgrade requests
3. Run the command: ``pip freeze > requirements.txt`` to regerate requirements file.
#### Any Background Context You Want To Provide?
When an event belonging to a particular category is created, all users interested in that category where not being sent calendar invites because the ```send_calendar_invites``` async function would fail silently. 
#### How can this be manually tested?
1. Fetch this branch (``bg-fix-get-invite-163719849``) to your local machine
2. Checkout to the branch newly fetched branch
3. Join a category that the default user belongs to. The default user belongs to the games category.
4. From the database table(auth_user), change the default user's email(Mayowa's) to yours such that the calendar invites are sent to you when an event is created in the games category.
5. Since calendar invites are only sent when the system is operating in production it will be necessary to temporarily copy line 74 and line 75 of ```/server/graphql_schemas/utils/helpers.py``` to line 71
5. Create an event in the games category slated for a later date than the current date
6. You should be able to recieve a calendar invite after step 6
#### What are the relevant pivotal tracker stories?
[#163719849 ](https://www.pivotaltracker.com/story/show/163719849)
#### Screenshot(s)
N/A
